math/maxima/Portfile

```diff
@@ -19,30 +18,60 @@ long_description \
         tensors. Maxima yields high precision numeric results \
         by using exact fractions, arbitrary precision integers, \
         and variable precision floating point numbers. Maxima \
-        can plot functions and data in two and three dimensions.
+        can plot functions and data in two and three dimensions. \
+        Maxima includes the following commands: \
+        * \"rmaxima\" is a front end which provides advanced line-editing \
+        facilities via rlwrap. \
+        * \"xmaxima\" is a graphical interface for maxima, written in Tcl/Tk.
 
 homepage        http://maxima.sourceforge.net/
-master_sites    sourceforge
 
-checksums       md5     3bccafb512ff1054be700c61601088af \
-                sha256  f58ff98013fcaa37d8661b941672c38db58f982f082d6cfcca1f1b5fac7d71b3 \
-                rmd160  3c4b153554c9b9cb3442e030d503fbe975a614fc
+subport maxima-devel {
+    conflicts   maxima
+
+    # git describe --tags : branch-5_41-base-415-g7e71fc4
+    # Date:  Fri Jun 15 17:26:53 2018 -0500
+    # commit 7e71fc4b6a95cab6418227754f01fc44ac845da1
+    version     5.41-dev-20180615
+    fetch.type  git
+    git.url     https://git.code.sf.net/p/maxima/code
+    git.branch  7e71fc4b6a95cab6418227754f01fc44ac845da1
+
+    use_autoreconf  yes
+
+    livecheck.type  none
+}
+
+if {${subport} eq ${name}} {
+    conflicts   maxima-devel
+
+    # get the source tarball from sourceforge.
+    master_sites    sourceforge:project/maxima/Maxima-source/${version}-source
+
+    checksums   size    41540226 \
+                sha256  daf8254600f4c7d9f079a0084178aaf036bd97a63b1e77b945f4120d7e401674 \
+                rmd160  e58cedfa395ac5cb4715d1b60ed7cf6ae48979cf
+
+    livecheck.regex     {<title>.*/Maxima-source/(.*)-source/maxima.*</title>}
+}
 
 depends_lib     port:sbcl
-depends_run     port:recode \
+depends_build   port:texinfo
```

Comment:
Please move these `depends_*` up or use `depends_build-append` here to avoid overriding the build dependencies `use_autoreconf` has set.